packages/docs/src/components/DocsHeadline.vue

```diff
@@ -4,7 +4,7 @@
       tag="span"
       inline
       :value="textComputed"
-    />
```

Comment:
Redundant symbol here and below.

Reply:
![image](https://user-images.githubusercontent.com/29167241/180735159-61b63172-48b0-4b8f-9127-6da61c874cf6.png)

Reply:
It's not redundant, it's added purposely.